app/concepts/matestack/ui/core/q/q.haml

```diff
@@ -0,0 +1,3 @@
+%q{@tag_attributes}
+  - if block_given?
```

Comment:
I would love this component to have a `options[:text`, like e.g. the `paragraph` or `span` component. Could you implement (and test & document) this?